ert3/config/_ensemble_config.py

```diff
@@ -14,7 +14,7 @@ class _EnsembleConfig(BaseModel):
 
 class ForwardModel(_EnsembleConfig):
     stages: List[str]
-    driver: Literal["local"] = "local"
+    driver: Literal["local", "pbs"] = "local"
```

Comment:
Do we want to test a config with driver set to `pbs`? e.g. in `test_ensemble_config:test_input`?

There's a balance between testing pydantic of course, but I think adding such a test would test our configuration setup - which makes sense.

Reply:
Done